pkg/transformers/openshift/openshift.go

```diff
@@ -19,50 +14,21 @@ import (
 	_ "github.com/openshift/origin/pkg/template/api/install"
 )
 
-func Transform(paths ...string) (*templateapi.Template, error) {
+func Transform(paths ...string) (*kapi.List, error) {
 
 	template, err := dockercompose.Generate(paths...)
 
 	if err != nil {
 		return nil, err
 	}
 
-	// Convert template objects to versioned objects
-	var convErr error
-	template.Objects, convErr = convertToVersion(template.Objects, "v1")
-	if convErr != nil {
-		panic(convErr)
-	}
+	list := ConvertToList(template)
```

Comment:
:+1: for normalizing the return types to a common type for both ``openshift`` and ``kubernetes``